lib/container_orchestrator.rb

```diff
@@ -1,28 +1,84 @@
+require 'kubeclient'
+
 class ContainerOrchestrator
```

Comment:
We may want to namespace this under ManageIQ, so as not to confuse any code from the container providers